src/main/java/com/amazon/dlic/auth/ldap/backend/LDAPAuthenticationBackend.java

```diff
@@ -54,6 +54,7 @@ public class LDAPAuthenticationBackend implements AuthenticationBackend {
 
     private final Settings settings;
     private final Path configPath;
+    private final boolean followReferrals;
```

Comment:
I like to name my booleans as questions where true/false has a clear answer, consider `followReferrals` -> `shouldFollowReferrals`